src/pkg/manifests/package:pkg:depot.p5m

```diff
@@ -68,3 +68,4 @@ license cr_Oracle license=cr_Oracle
 depend type=require fmri=web/server/apache-24
 # p5p support in the system repository requires mod_wsgi
 depend type=require fmri=web/server/apache-24/module/apache-wsgi-35
+depend type=require fmri=web/server/apache-24/module/apache-wsgi-39
```

Comment:
why should depot need two python versions?

Reply:
It does not need them, but as both can co-exist depot will use the version it is compiled against. I am copying the setup that all other packages use at this moment. Once the transition to 3.9 is completed in userland 3.5 gets dropped.

Reply:
should it be require-any type of dependency?

Reply:
No, it must not be.
We have mod_wsgi as mediated link, thus require-any would cause that to fail.
```
link mediator=python mediator-version=3.5 path=usr/apache2/2.4/libexec/mod_wsgi.so target=mod_wsgi-3.5.so
```
With both dependencies mod_wsgi is linked as expected to the mediated python version. I don't know if dependency actions support mediator properties cleanly.

Reply:
Then you probably wants conditional dependency with proper predicate.  Something similar like perl modules do.   For example: `pkg contents -mr file-chdir | grep ^depend`.

Reply:
Ah, thanks. Yeah that make sense. Updated.